lib/rust/lexer/definition/src/lexer.rs

```diff
@@ -227,25 +431,43 @@ impl EnsoLexer {
         }
     }
 
+    /// Triggered when an operator is immediately followed by another (e.g. `.+` or `. ==`).
```

Comment:
this docs are talking about other situation than the function name. For example, `+ -` this is also dotted operator according to docs